src/bin/driver.rs

```diff
@@ -549,13 +588,20 @@ mod app {
         });
     }
 
-    // Task for waiting the relay transition times.
-    #[task(priority = 1, shared=[relay_state])]
-    fn handle_relay(mut c: handle_relay::Context, channel: driver::Channel) {
-        let delay = (c.shared.relay_state)
-            .lock(|state| state[channel as usize].handle_relay());
+    // Task for handling the Powerup/-down sequence
+    #[task(priority = 1, shared=[output_state, settings])]
```

Comment:
What if both outputs are enabled almost simultaneously. The task queue may overflow and the `spawn_after()` will `Err`, `unwrap` and `panic`.

Reply:
Yes, I'm aware that currently the outputs cannot be enabled/disabled simultaneously. 
I didn't set the capacity to 2 yet because I wanted things to panic when things go wrong with one channel and we try to spawn another handle.
But I would say just setting the capacity to 2 is fine later.